src/System.Threading.Thread/tests/ThreadTests.cs

```diff
@@ -331,12 +331,27 @@ public static void ApartmentState_NoAttributePresent_STA_Unix()
         [Fact]
         [SkipOnTargetFramework(TargetFrameworkMonikers.NetFramework)]
         [SkipOnTargetFramework(TargetFrameworkMonikers.Mono)]
-        public static void CurrentCultureTest_SkipOnDesktopFramework()
+        [SkipOnTargetFramework(TargetFrameworkMonikers.Uap)]
+        public static void CurrentCultureTest_DifferentThread()
```

Comment:
Why is this skipped on NETFX?

Reply:
Ah I see discusion in issue. Is it worth having a test for NETFX specifically, to document it's behavior?

Reply:
I think that writing NetFX-specific tests in CoreFX is waste of energy in the long run. I expect that we are going to stop maintaining and running tests on netfx in CoreFX at some point in future because of they are not going to provide value anymore.

I will make the comments on the existing test better.